addons/hr_expense/models/product_template.py

```diff
@@ -7,18 +7,18 @@
 class ProductTemplate(models.Model):
     _inherit = "product.template"
 
+    @api.model
+    def default_get(self, fields):
+        result = super(ProductTemplate, self).default_get(fields)
+        if self.env.context.get('default_can_be_expensed'):
+            result.update({
+                'supplier_taxes_id': False,
+            })
```

Comment:
`update({})` is many order of magnitudes slower than straight assignation :innocent: 

```suggestion
            result['supplier_taxes_id'] = False
```

![image](https://user-images.githubusercontent.com/50135801/151961529-9ff39d5a-1639-4379-8a31-bef446bd5dfe.png)
![image](https://user-images.githubusercontent.com/50135801/151961514-4c331aa1-da0f-4ac6-932b-2c2d090c6682.png)

Reply:
Done ^^